music_assistant/server/providers/snapcast/__init__.py

```diff
@@ -130,6 +134,22 @@ async def get_config_entries(
             hidden=not local_snapserver_present,
             help_link="https://raw.githubusercontent.com/badaix/snapcast/86cd4b2b63e750a72e0dfe6a46d47caf01426c8d/server/etc/snapserver.conf",
         ),
+        ConfigEntry(
+            key=CONF_SERVER_CHUNK_MS,
+            type=ConfigEntryType.INTEGER,
+            range=(10, 100),
+            default_value=26,
+            label="Snapserver chunk size",
+            description="Default source stream read chunk size [ms]. "
+            "The server will continuously read this number of milliseconds from the source into "
+            "buffer and pass this buffer to the encoder."
+            "The encoded buffer is sent to the clients. Some codecs have a higher"
+            "latency and will need more data, e.g. Flac will need ~26ms chunks",
+            required=False,
+            category="Built-in Snapserver Settings",
```

Comment:
let's not use a full text here. Use an identifier and add it to the translations on the frontend.

Reply:
Where can I find an example of this, I searched in other suppliers and for now this is how it was done before

Reply:
I also checked the server and frontend repo, not sure how do I make it an identifier...

Reply:
In the frontend is to add a line to en.js, isn't it ?

Reply:
> In the frontend is to add a line to en.js, isn't it ?

I think that JSON file is autogenerated from Lokalise i18n platform and should not be modified by hand. We need to add a "key" instead of a naive string in the server code somewhere, then put that key into Lokalise and provide translations to it. Still figuring out how to do that

Reply:
Okay I think I figured out what to do...
![image](https://github.com/user-attachments/assets/22922b26-4c2f-4db9-ab8f-06febda28bcb)

Reply:
I think that permissions are required, I don't have them in lokalise, the option that is in the docs doesn't appear to me

Reply:
#1693 I have removed the description and separated some of the hard coded values

Reply:
category should be a (string) key only. All lowercase, no spaces.
The default keys are generic and advanced. If you want to add more, then the key needs to be in the translations/en.json file from frontend:

https://github.com/music-assistant/frontend/blob/main/src/translations/en.json#L291

Only edit the en.json file, the others are maintained by lokalize.

Reply:
> category should be a (string) key only. All lowercase, no spaces. The default keys are generic and advanced. If you want to add more, then the key needs to be in the translations/en.json file from frontend:
> 
> https://github.com/music-assistant/frontend/blob/main/src/translations/en.json#L291
> 
> Only edit the en.json file, the others are maintained by lokalize.

Got it, I'll change it later as I am a little busy today.